openflow_input/bsn_port_desc_module_eeprom

```diff
@@ -0,0 +1,228 @@
+// Copyright 2018, Big Switch Networks, Inc.
+//
+// LoxiGen is licensed under the Eclipse Public License, version 1.0 (EPL), with
+// the following special exception:
+//
+// LOXI Exception
+//
+// As a special exception to the terms of the EPL, you may distribute libraries
+// generated by LoxiGen (LoxiGen Libraries) under the terms of your choice, provided
+// that copyright and licensing notices generated by LoxiGen are not altered or removed
+// from the LoxiGen Libraries and the notice provided below is (i) included in
+// the LoxiGen Libraries, if distributed in source code form and (ii) included in any
+// documentation for the LoxiGen Libraries, if distributed in binary form.
+//
+// Notice: "Copyright 2013, Big Switch Networks, Inc. This library was generated by the LoxiGen Compiler."
+//
+// You may not use this file except in compliance with the EPL or LOXI Exception. You may obtain
+// a copy of the EPL at:
+//
+// http://www.eclipse.org/legal/epl-v10.html
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// EPL for the specific language governing permissions and limitations
+// under the EPL.
+#version 5
+#version 6
+
+enum ofp_bsn_unit_type(wire_type=uint8_t, bitmask=False) {
+    OFP_BSN_UNIT_NONE = 0x0,
+    OFP_BSN_UNIT_AMP = 0x2,
+    OFP_BSN_UNIT_VOL = 0x3,
+    OFP_BSN_UNIT_KM = 0x4,
+    OFP_BSN_UNIT_M = 0x5,
+    OFP_BSN_UNIT_S = 0x6,
+    OFP_BSN_UNIT_C = 0x7,
+    OFP_BSN_UNIT_F = 0x8
+};
+
+struct of_bsn_unit {
+    uint32_t value;
+    /* We could probably use an enum to convey the unit */
```

Comment:
suggest updating this placeholder comment

Reply:
Ok sure.